fastlane/lib/fastlane/helper/sh_helper.rb

```diff
@@ -35,7 +39,9 @@ def self.sh_no_action(command, log: true)
 
         if exit_status != 0
           # this will also append the output to the exception
-          if log
+          if print_command
+            message = "Exit status of command '#{command}' was #{exit_status} instead of 0."
+          elsif print_command && print_command_output
```

Comment:
We should probably handle the third case, `print_command_output` but not  `print_command` (in which case we still want to print the result)